src/orquestra/sdk/_base/_api/_config.py

```diff
@@ -58,7 +58,7 @@ def __init__(
                 "`RuntimeConfig.ce()` for Compute Engine. \n"
             )
 
-        self._name = name
+        self._name: str = name
```

Comment:
**question**: Is this type hint needed? I'd think type checkers infer the types of attributes from the init's argument types